plugins/woocommerce/includes/class-wc-frontend-scripts.php

```diff
@@ -267,6 +267,7 @@ private static function register_scripts() {
 				'src'     => self::get_asset_url( 'assets/js/frontend/checkout' . $suffix . '.js' ),
 				'deps'    => array( 'jquery', 'woocommerce', 'wc-country-select', 'wc-address-i18n' ),
 				'version' => $version,
+				'args'    => true,
```

Comment:
Nit-pick: You could use the following format to follow the new WP 6.3+ convention. It would still be compatible with older WordPress versions, as it will evaluate to `true` when casting to a boolean.
```suggestion
				'args'    => array( 'in_footer' => true ),
```

Reply:
If this is safe for WP 6.3+, we can adopt it without worrying about older versions (as of WooCommerce 8.3, we require a minimum of WP 6.3).

Reply:
It is 👍

Reply:
Thanks @felixarntz !